# gaze_direction.msg
float64 predicted_gaze_direction
